### ISSUE TITLE
Getting Started With Axios In Nuxt

### ISSUE THREAD

**Issue by Ghassany** [section-engineering-education/engineering-education#5529]
### Topic Suggestion
Getting Started With Axios In Nuxt

#### Proposed article introduction
Nuxt.js provides an Axios module for easy integration with your application. Axios is a promise-based HTTP client that works in the browser and Node.js environment or, in simpler terms, it is a tool for making requests (e.g API calls) in client-side applications and Node.js environment.

In this tutorial, we’re going to learn how to use the Axios module and how to make a request on the server-side using asyncData and fetch. These two methods make a request on the server-side but they have some differences which we’re also going to cover. Finally, we’ll learn how to perform authentication and secure pages/routes using the auth module and auth middleware.

#### Key Takeaways

- What Is The Nuxt.Js Axios Module?
- Making Your First Request Using Axios
- Introduction To The Auth Module
- Using The Auth Module 

#### Article quality
In this tutorial, we will learn how to make a request in our Nuxt.js applications using the Axios module. Finally, we will learn how to add authentication to our application using the Auth module.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

**Comment by ahmadmardeni1** [comment]
Thank you for submitting your topic. @Ghassany 
After some careful consideration, it struck us that this topic may be a bit over-saturated throughout other blog sites and official documentation.

We believe this is the best way for students to build a great portfolio (for potential employers) is by building what does not exist and what can provide the most value.

Please feel free to suggest an alternate topic to explore. 🚀
